download_and_create_reference_datasets/v02/mito/utils.py

```diff
@@ -0,0 +1,99 @@
+import argparse
+import logging
+import json
+import tqdm
+import tempfile
+import os
+import zipfile
+import requests
+
+import hail as hl
+
+logging.basicConfig(format='%(asctime)s %(levelname)-8s %(message)s', level='INFO')
+logger = logging.getLogger(__name__)
+
+contig_recoding={'1': 'chr1', '10': 'chr10', '11': 'chr11', '12': 'chr12', '13': 'chr13', '14': 'chr14', '15': 'chr15',
+ '16': 'chr16', '17': 'chr17', '18': 'chr18', '19': 'chr19', '2': 'chr2', '20': 'chr20', '21': 'chr21', '22': 'chr22',
+ '3': 'chr3', '4': 'chr4', '5': 'chr5', '6': 'chr6', '7': 'chr7', '8': 'chr8', '9': 'chr9', 'X': 'chrX', 'Y': 'chrY',
+ 'MT': 'chrM', 'NW_009646201.1': 'chr1'}
```

Comment:
Good questions. I just got the recoding from Hail and I don't fully understand it. It can be per dataset based. It works for the Clinvar dataset. But, now, we don't need to combine the Clinvar dataset because we decided to share them with the SNPs. So, I'm going to remove the `vcf` input type and remove the recoding as well.